vendor.conf

```diff
@@ -1,5 +1,3 @@
-github.com/containers/image
-
```

Comment:
These two lines are not a self-vendoring, and intentional — IIRC `trash` prefers them to be there. Or has that changed?

Reply:
I've dropped this change